finagle-core/src/main/scala/com/twitter/finagle/loadbalancer/RoundRobin.scala

```diff
@@ -0,0 +1,105 @@
+package com.twitter.finagle.loadbalancer
+
+import com.twitter.finagle.{ClientConnection, NoBrokersAvailableException, Service,
+  ServiceFactory, ServiceFactoryProxy, Status}
+import com.twitter.finagle.service.FailingFactory
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle.util.OnReady
+import com.twitter.util.{Activity, Future, Promise, Time}
+
+/**
+ * A simple round robin balancer that chooses the next backend in
+ * the list for each request.
+ */
+class RoundRobinBalancer[Req, Rep](
+  protected val activity: Activity[Traversable[ServiceFactory[Req, Rep]]],
```

Comment:
the convention of the other balancers is to make these protected. I don't know why that choice was made but it's not strictly required so I've reverted back to default access modifiers for RoundRobinBalancer